doc/manual/src/contributing/hacking.md

```diff
@@ -108,3 +108,72 @@ These tests include everything that needs to interact with external services or
 Because these tests are expensive and require more than what the standard github-actions setup provides, they only run on the master branch (on <https://hydra.nixos.org/jobset/nix/master>).
 
 You can run them manually with `nix build .#hydraJobs.tests.{testName}` or `nix-build -A hydraJobs.tests.{testName}`
+
+## Testing the install scripts
```

Comment:
To make it consistent with the other sections:

```suggestion
## Installer tests
```

Reply:
adopted change locally, available when I push